apps/nuxt3-ssr/server/routes/[schema]/api/file/[...].ts

```diff
@@ -0,0 +1,8 @@
+import { joinURL } from "ufo"; 
+
```

Comment:
this file has a very strange name

Reply:
this a wildcard (https://nuxt.com/docs/guide/directory-structure/server#catch-all-route)

Reply:
done